lms/templates/view.html

```diff
@@ -39,7 +39,10 @@ <h1>תצוגת תרגיל {{ solution['exercise']['id'] }}: {{ solution['exercis
   <div id="view-body">
     <div class="code-view-container">
       <div id="code-view" data-id="{{ solution['id'] }}" data-file="{{ current_file.id }}" data-exercise="{{ solution['exercise']['id'] }}" data-role="{{ role }}">
-        <pre><code class="language-python line-numbers highlight">{{- current_file.code | trim(chars=' ') | e -}}</code></pre>
+        <pre><code id="python-user-code" class="language-python line-numbers highlight">{{- current_file.code | trim(chars=' ') | e -}}</code></pre>
+      </div>
+      <div id="copy-code">
+        <button class="fa fa-copy" id="copy-button" title="Copy code" onclick="copy_code()" style="background-color:white;font-size:20px;"></button>
```

Comment:
The style should be in the CSS, and the onclick should be in the JS file.

We try not to use inline CSS/JS.